src/api/kube.ts

```diff
@@ -1404,6 +1404,52 @@ export class KubeHelper {
     }
   }
 
+  async getOAuthClientAuthorizations(clientName: string): Promise<string[]> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      const { body } = await customObjectsApi.listClusterCustomObject('oauth.openshift.io', 'v1', 'oauthclientauthorizations')
+
+      if (!body.items) {
+        return []
+      }
+      const oauthClientAuthorizations = body.items as any[]
+      return oauthClientAuthorizations.filter(o => o.metadata.name && o.clientName === clientName)
+    } catch {
+      return []
```

Comment:
Shouldn't we throw an error here?

Reply:
Not necessary. We can throw an error in case if for some reason is not possible to obtain `oauthClientAuthorizations`

Reply:
I meant a situation when `customObjectsApi.listClusterCustomObject` fails due to network or permission or so error. Is it safe to return empty list in such case?

Reply:
yes, it is. This functions will return `[]` . For example if there is some network or permissions the error will be thrown here: https://github.com/flacatus/chectl/blob/delete_cleanup/src/api/kube.ts#L1422-L1431

Reply:
Maybe we should at least log this error.